mllib/src/main/scala/org/apache/spark/mllib/classification/LogisticRegression.scala

```diff
@@ -223,6 +241,9 @@ class LogisticRegressionWithSGD private[mllib] (
    */
   def this() = this(1.0, 100, 0.01, 1.0)
 
+  /**
+   * @since 0.8.0
```

Comment:
We only put `@since` on public classes and methods. This is a package private method. So it is not necessary to add `@since` tag.